composer.json

```diff
@@ -25,7 +25,7 @@
         "illuminate/contracts": "^7.0",
         "illuminate/database": "^7.0",
         "illuminate/support": "^7.0",
-        "laravie/query-filter": "dev-master"
+        "laravie/query-filter": "^1.5"
```

Comment:
it's not determined yet whether master will be 1.5.

This branch is targeted for Laravel 7 and wouldn't be merged or have a plan release until Laravel 7 is close to being ready.

Reply:
Well, the master branch currently has code in it that says deprecated in 1.5.0. ;)

Reply:
https://github.com/laravie/query-filter/blob/1.x/src/Value/Keywords.php

`1.x` branch also has that

Reply:
Ok, well, my point was actually that 1.x has that, indicating 1.5.0 next minor release. :P

Reply:
yeah, but only master support installation for Laravel 7, not 1.x

Reply:
Ah, right. So 2.x?

Reply:
Should the version be `^2.0` maybe?

Reply:
I don't know yet.

Basically for any project that I have `master` without branch-aliases, that simply mean I prepared the package to be installed in upcoming version (mostly upcoming Laravel). It depends on how big of a change the next version will be when the next version is ready then I would either merge master to 1.x or release master as 2.0.

At the same time, if there a new for a major breaking change in 1.x, then I will branch it out to 2.x and the version for master will move around too.